src/app.js

```diff
@@ -22,11 +22,9 @@ class App extends Component {
 			mode
 		} = this.props;
 
-		// app app2 app3
-
 		return (
 			<div>
-				<Button classNames={classNames('app', 'app2', {'app3': mode === 'light'})} value="Toggle Styles" label="Toggle Styles" action={this.toggleStyles}/>
+				<Button classNames={classNames( mode == 'light' ? 'appLight' : 'appDark')} value="Toggle Styles" label="Toggle Styles" action={this.toggleStyles}/>
```

Comment:
Rather than passing strings as arg(s) to that function it also accepts objects which in my opinion are more useful than using strings in ternary situations. Try this:

`classNames({'theclassname': aBoolean})` so you can have your checks as the value and that will conditional place them in there based on if the expression evaluates to true or not

Reply:
`classNames({'appLight': mode === 'light', 'appDark': mode === 'dark'}). Also use `===` in general, the extra = checks type as well